src/test/regress/expected/gp_query_id.out

```diff
@@ -0,0 +1,1296 @@
+NOTICE:  START ProcessUtility | Q: vacuum analyze t; | QUERY ID: 115  (seg1 127.0.1.1:6003 pid=818837)
+NOTICE:  START ProcessUtility | Q: vacuum analyze t; | QUERY ID: 115  (seg2 127.0.1.1:6004 pid=818838)
+NOTICE:  START ExecutorStart | Q: SELECT tupcount, segno FROM pg_aoseg.pg_aoseg_17207 WHERE segno = 0 | QUERY ID: 115  (seg0 127.0.1.1:6002 pid=818836)
+NOTICE:  START ExecutorStart | Q: SELECT tupcount, segno FROM pg_aoseg.pg_aoseg_17207 WHERE segno = 0 | QUERY ID: 115  (seg1 127.0.1.1:6003 pid=818837)
+NOTICE:  START ExecutorStart | Q: SELECT tupcount, segno FROM pg_aoseg.pg_aoseg_17207 WHERE segno = 0 | QUERY ID: 115  (seg2 127.0.1.1:6004 pid=818838)
+NOTICE:  START ProcessUtility | Q: vacuum analyze t; | QUERY ID: 115  (seg0 127.0.1.1:6002 pid=818836)
+NOTICE:  START ProcessUtility | Q: vacuum analyze t; | QUERY ID: 115  (seg1 127.0.1.1:6003 pid=818837)
+NOTICE:  START ProcessUtility | Q: vacuum analyze t; | QUERY ID: 115  (seg2 127.0.1.1:6004 pid=818838)
+NOTICE:  START ExecutorStart | Q: SELECT tupcount, segno, state FROM pg_aoseg.pg_aoseg_17207 | QUERY ID: 115  (seg2 127.0.1.1:6004 pid=818838)
+NOTICE:  START ExecutorStart | Q: SELECT tupcount, segno, state FROM pg_aoseg.pg_aoseg_17207 | QUERY ID: 115  (seg0 127.0.1.1:6002 pid=818836)
+NOTICE:  START ExecutorStart | Q: SELECT tupcount, segno, state FROM pg_aoseg.pg_aoseg_17207 | QUERY ID: 115  (seg1 127.0.1.1:6003 pid=818837)
+NOTICE:  START ProcessUtility | Q: vacuum analyze t; | QUERY ID: 115  (seg0 127.0.1.1:6002 pid=818836)
+NOTICE:  START ProcessUtility | Q: vacuum analyze t; | QUERY ID: 115  (seg1 127.0.1.1:6003 pid=818837)
+NOTICE:  START ProcessUtility | Q: vacuum analyze t; | QUERY ID: 115  (seg2 127.0.1.1:6004 pid=818838)
+NOTICE:  START ExecutorStart | Q: SELECT tupcount, segno FROM pg_aoseg.pg_aoseg_17207 | QUERY ID: 115  (seg0 127.0.1.1:6002 pid=818836)
+NOTICE:  START ExecutorStart | Q: SELECT tupcount, segno FROM pg_aoseg.pg_aoseg_17207 | QUERY ID: 115  (seg1 127.0.1.1:6003 pid=818837)
+NOTICE:  START ExecutorStart | Q: SELECT tupcount, segno FROM pg_aoseg.pg_aoseg_17207 | QUERY ID: 115  (seg2 127.0.1.1:6004 pid=818838)
+NOTICE:  START ProcessUtility | Q: vacuum analyze t; | QUERY ID: 115  (seg0 127.0.1.1:6002 pid=818836)
+NOTICE:  START ProcessUtility | Q: vacuum analyze t; | QUERY ID: 115  (seg1 127.0.1.1:6003 pid=818837)
+NOTICE:  START ProcessUtility | Q: vacuum analyze t; | QUERY ID: 115  (seg2 127.0.1.1:6004 pid=818838)
+NOTICE:  START ExecutorStart | Q: select pg_catalog.pg_relation_size(17207, 'main') | QUERY ID: 115  (seg0 127.0.1.1:6002 pid=818836)
+NOTICE:  START ExecutorStart | Q: select pg_catalog.pg_relation_size(17207, 'main') | QUERY ID: 115  (seg1 127.0.1.1:6003 pid=818837)
+NOTICE:  START ExecutorStart | Q: select pg_catalog.pg_relation_size(17207, 'main') | QUERY ID: 115  (seg2 127.0.1.1:6004 pid=818838)
+NOTICE:  START ExecutorStart | Q: select pg_catalog.gp_acquire_sample_rows(17207, 10000, 'f'); | QUERY ID: 115  (seg0 127.0.1.1:6002 pid=818836)
```

Comment:
Is it ok that query id is the same for the different queries?

Reply:
Yes, because these are auxiliary queries related to the same user's query (`vacuum` in this case):
```
vacuum analyze t;
NOTICE:  START ProcessUtility | Q: vacuum analyze t; | QUERY ID: 115
...
```
It is what currently expected by monitoring extension - you can check vacuum in `cdb_dispatch_replace_ccnt` test in the adcc extension.